mandatory/2-function-creation.js

```diff
@@ -15,26 +17,46 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  return totalPrice < 200 ? (totalPrice - totalPrice * 0.05) : (totalPrice - totalPrice * 0.1);
```

Comment:
This line starts to get too long, it's usually better to keep lines short even if it means code needs to be cut over multiple lines.

Some other strategies to use to shorten the line:
- `totalPrice * (1-0.05)` - keeps the 5% in the code as required by the spec
- `totalPrice * 0.95` - although this removes the 5% in the code in a clear way (it now has 95%), based on what kind of code people are writing this might still be okay